### PR TITLE
Refactor Next Checkout Payment Page to focus on SE flow

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Refactor Next Checkout Payment page to focus on Swedish flow.

Remove Adyen-specific setup (but keep the React components for later).

Add new env var: `NEXT_PUBLIC_ORIGIN_URL` to be able to generate redirect URL after payment is connected.

Update logging to include relevant contextual values.

## Justify why they are needed

Since we are planning to go live in Sweden - let's focus on that for now.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #744** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/744" target="_blank">Graphite</a> 
    * **PR #747** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/747" target="_blank">Graphite</a> 
      * **PR #748** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/748" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/748?utm_source=stack-comment).

File: apps/store/src/utils/PageLink.ts [hunk 1]
```diff
@@ -1,6 +1,11 @@
 import { AuthStatus } from '@/components/CheckoutPaymentPage/CheckoutPaymentPage.constants'
 import { RoutingLocale } from '@/utils/l10n/types'
 
+const ORIGIN_URL =
```
Comment (alebedev) [inline apps/store/src/utils/PageLink.ts:4]: Would we ever need to overwrite Vercel's variable?

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/utils/PageLink.ts (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/utils/PageLink.ts:4]: Yes, we don't handle domains through Vercel so we need to override it for any deployments that we serve through CloudFront :)

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/748?utm_source=stack-comment).


Job Reference: ozJVRlc8vbieEifF13OS

**Pull request merged**